packages/block-editor/src/components/image-editor/index.js

```diff
@@ -6,11 +6,11 @@ import { ToolbarGroup, ToolbarItem } from '@wordpress/components';
 /**
  * Internal dependencies
  */
+import AspectRatioDropdown from '../aspect-ratio-dropdown';
```

Comment:
What was the reason for extracting this in a separate component? It seems to be used only by the image editor.

Reply:
That is a very good question. This was already done before I took over the branch. 🤔

Reply:
One reason I think we should group it with the Image Editor is that the dropdown also resizes the image on the server. Having it as a stand-alone component could be misleading.